tests/reduction/reducer_api_fp64.cpp

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright (c) 2023 The Khronos Group Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+*******************************************************************************/
+
+#include "../../util/extensions.h"
+#include "../common/common.h"
+#include "reducer_api.h"
+
+#include <string>
+
+TEST_CASE("reducer api fp64", "[reducer][fp64]") {
+  sycl::queue queue = sycl_cts::util::get_cts_object::queue();
+  using avaliability = sycl_cts::util::extensions::availability<
+      sycl_cts::util::extensions::tag::fp64>;
+  if (!avaliability::check(queue)) {
+    SKIP(
+        "Device does not support double precision floating point operations. "
+        "Skipping the test case.");
```

Comment:
```suggestion
        "Device does not support double precision floating point operations. ");
```
I think `SKIP` macro output makes it clear that the test is skipped, so there is no need to say it in the message. @psalz, am I right?

Reply:
That indeed looks to be the case, I have made the change as you suggest.